From: Dmitry Bogatov <KAction@gnu.org>
Date: Tue, 12 Mar 2019 19:07:02 +0000
Subject: Make /etc/runit/2 respect FHS

Last-Update: 2016-05-31

On Debian system services directory is /etc/service, not /service.
Last-Update: 2016-05-31
---
 runit-2.1.2/etc/2 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/2 b/etc/2
index 3bb822d..7c6aab3 100755
--- a/etc/2
+++ b/etc/2
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-PATH=/command:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
+PATH=/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
 
 exec env - PATH=$PATH \
-runsvdir -P /service 'log: ...........................................................................................................................................................................................................................................................................................................................................................................................................'
+runsvdir -P /etc/service 'log: ...........................................................................................................................................................................................................................................................................................................................................................................................................'
